docs/virtual_machines/accessing_virtual_machines.md

```diff
@@ -290,21 +218,168 @@ spec:
           propagationMethod:
             qemuGuestAgent:
               users:
-              - "fedora"
+              - fedora
       volumes:
-      - dataVolume:
-          name: fedora-dv
-        name: disk0
+      - containerDisk:
+          image: quay.io/containerdisks/fedora:latest
+        name: containerdisk
       - cloudInitConfigDrive:
-          userData: |
-            #!/bin/bash
+          userData: |-
+            #cloud-config
+            password: fedora
+            chpasswd: { expire: False }
+            # Disable SELinux for now, so qemu-guest-agent can write the authorized_keys file
+            # The selinux-policy is too restrictive currently, see open bugs:
+            #   - https://bugzilla.redhat.com/show_bug.cgi?id=1917024
+            #   - https://bugzilla.redhat.com/show_bug.cgi?id=2028762
+            #   - https://bugzilla.redhat.com/show_bug.cgi?id=2057310
+            bootcmd:
+              - setenforce 0
+        name: cloudinitdisk
+EOF
+```
+
+### Accessing SSH with virtctl
+
+Every `VirtualMachineInstance` provides a `virtualmachineinstances/portforward`
+subresource, that can be used to create a websocket backed network tunnel to
+a port inside the instance similar to Kubernetes pods.
```

Comment:
I personally prefer if the sentence's subject is the user, rather than the VMI. Something like:
```suggestion
The user can create a websocket backed network tunnel to
a port inside the instance by using the VirtualMachineInstance virtualmachineinstances/portforward
subresource.
```